templates/engage/case-study-gmo-pepabo.html

```diff
@@ -0,0 +1,44 @@
+{% extends "_base/base.html" %}
+
+{% block title %}GMOペパボについて{% endblock %}
+
+{% block meta_description %}GMOペパボがカーネルのライブパッチによって運用コストと2,500時間の手作業を削減{% endblock %}
+
+{% block meta_image %}{% endblock %}
```

Comment:
This is missing the meta image given [here](https://github.com/canonical-web-and-design/web-squad/issues/3055#issuecomment-676479694) - you may need to upload it to the asset manager, not sure if it already has been.